### PR TITLE
 [FE][PT][FX] Add Support for torch.ops.aten.bernoulli.p 

### PR DESCRIPTION
**Pull request by anzr299**
### Details:
With the latest conformance test runs, it was found that the operator `torch.ops.aten.bernoulli.p` was causing graph breaks due to being unsupported. This PR aims to add support for the same.

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_bernoulli.py [hunk 1]
```diff
@@ -90,6 +98,7 @@ def bernoulli_with_p(self, input):
     @pytest.mark.parametrize("seed", [12])
     @pytest.mark.nightly
     @pytest.mark.precommit
```
Comment (rkazants) [inline tests/layer_tests/pytorch_tests/test_bernoulli.py:101]: can we also add `@pytest.mark.precommit_export`

Comment (anzr299) [reply tests/layer_tests/pytorch_tests/test_bernoulli.py:101]: I noticed that it had some issues tracing the generator instances.

Comment (rkazants) [reply tests/layer_tests/pytorch_tests/test_bernoulli.py:101]: Yeah, I recalled that I met this issue.

File: tests/layer_tests/pytorch_tests/test_bernoulli.py [hunk 2]
```diff
@@ -68,10 +73,13 @@ class aten_bernoulli(torch.nn.Module):
...
                 super().__init__()
                 gen = torch.Generator()
-                gen.manual_seed(seed)
                 self.gen = gen
                 self.p = p
+                self.seed = seed
                 self.forward = self.bernoulli_with_p
+            def __call__(self, *args, **kwargs):
+                super().__call__(*args, **kwargs)
+                self.gen.manual_seed(self.seed)
```
Comment (rkazants) [inline tests/layer_tests/pytorch_tests/test_bernoulli.py]: why do we need it? Why `forward` method is not sufficient? It can be a case that you trace a different function that does not include `torch.bernoulli`

Comment (anzr299) [reply tests/layer_tests/pytorch_tests/test_bernoulli.py]: calling manual seed in forward method causes error with tracing

**Review by mvafin — APPROVED** [review]
